### DISCUSSION TITLE
Approved but package is not accessible

### DISCUSSION THREAD

**Discussion by Kenneth6589** [community/community#56793] [GitHub Education]
### Hi everyone,

I verified my account and it approved immediately but I can't explore the student pack
![Screenshot_2023-05-30-21-19-28-804_com brave browser](https://github.com/community/community/assets/134955096/8fc0b35b-eb22-4084-ad33-4c5c1160840a)

**Comment by Kenneth6589** [comment]
How do I solve this?

**Reply by patty111** [reply]
https://education.github.com/pack#offers

**Comment by samus-aran** [comment]
Hi @Kenneth6589  👋!

We've updated our application and re-verification process to ensure that GitHub and our partners can offer exceptional, free benefits to students worldwide during their academic journey.

With this update, when you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

It's great to have you in the GitHub Education community and we hope to see you around 👋
